config/Betterloadingscreen/tips/en_US.txt

```diff
@@ -15,6 +15,7 @@ Adding water to an empty but hot Boiler will make a loud bang!
 Avoid Eldritch Obelisks!
 You can bind the questbook to a key to save you from using the item
 Pam's Gardens can be picked up with right-click
+Dont make the LV Extruder!
```

Comment:
Sure the extruder is not very usefull but for thinkers tool they are when you slowly progress .